openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/MlModelSummary/MlModelSummary.component.tsx

```diff
@@ -29,11 +34,13 @@ import { BasicEntityInfo } from '../SummaryList/SummaryList.interface';
 interface MlModelSummaryProps {
   entityDetails: Mlmodel;
   componentType?: string;
+  tags?: (TagLabel | undefined)[];
```

Comment:
is this the expected type? is it possible to get undefined in an array? Please double check